VultisigApp/VultisigApp/Views/Send/SendCryptoDetailsView.swift

```diff
@@ -247,6 +283,10 @@ struct SendCryptoDetailsView: View {
         await BalanceService.shared.updateBalance(for: tx.coin)
         coinBalance = tx.coin.balanceString
     }
+    
+    private func showMemo() -> Bool {
+        tx.coin.isNativeToken && tx.coin.tokenSchema != "ERC20"
```

Comment:
`tx.coin.tokenSchema != "ERC20"` is not required , because ERC20 will not be a native token